src/CLI/clitree/cli-xml/sonic_types.xml

```diff
@@ -53,7 +53,7 @@ limitations under the License.
     <PTYPE
         name="LAG_ID"
         method="integer"
-        pattern="1..256"
+        pattern="1..9999"
```

Comment:
We cant just make this change without ensuring that every command that prints the interface name in the output works correctly, especially the ones that have column output.

Reply:
Hi @joyas-joseph I did try "show interface PortChannel" and "show interface PortChannel 9876". Both are displaying non-table format.
The 'show interface status', 'show interface counters' are displaying data in tabular format however, they are displaying ethernet interfaces.
Other than this, I dont see any other command that is displaying name of an interface that is port channel.. The width for the name seem to be 15 which is sufficient for the maximum "PortChannel9999"

Reply:
Changed two renderers

Reply:
Please check show portchannel summary output as well.